docs/UserGuide.md

```diff
@@ -270,16 +282,17 @@ If your changes to the data file makes its format invalid, InternConnect will di
+| **Help**     | `help`                                                                                                                                                                                                                                                                            |
+| **Add**      | `add {mandatory_field_parameter/FIELD_DETAIL} [t/TAGS]...` <br> e.g., `add n/John Doe p/98765432 e/johnd@example.com a/311, Clementi Ave 2, #02-25 c/3.50/4.00 g/male u/Nanyang Polytechnic gd/05-2024 m/Computer Science ji/173296 jt/Software Engineer Intern t/rejected t/KIV` |
+| **List**     | `list`                                                                                                                                                                                                                                                                            | 
+| **View**     | `view INDEX`<br> e.g., `view 2`                                                                                                                                                                                                                                                   |
+| **Edit**     | `edit INDEX [parameter/NEW_PARAMETER_DETAIL]...`<br> e.g.,`edit 1 p/91234567 e/bob@example.com`                                                                                                                                                                                   |
+| **Delete**   | `delete INDEX`<br> e.g., `delete 3`                                                                                                                                                                                                                                               |
+| **Find**     | `find KEYWORD [MORE_KEYWORDS]`<br> e.g., `find James Jake`                                                                                                                                                                                                                        |
+| **Import**   | `import FILE_PATH`<br> e.g., `import nus_students.json`                                                                                                                                                                                                                           | 
+| **Export**   | `export`                                                                                                                                                                                                                                                                          |
+| **Checkout** | `checkout 27-oct-2022`                                                                                                                                                                                                                                                            |
```

Comment:
To standardize the format, please use `checkout FILE_NAME` <be> e.g., `checkout 27-oct-2022`

Reply:
Line changed to `| **Checkout** | checkout FILE_NAME `